app/models/game.rb

```diff
@@ -54,7 +54,6 @@ def clear_board!
 
   # Switch turns to the opposing user
   def update_turn
-    puts "UPDATING TURN!"
     self.turn == self.white_player_id ? self.turn = self.black_player_id : self.turn = self.white_player_id
```

Comment:
no need for 'self.' notation here.. you can just use turn, white_player_id etc.. using attr_accessors gives you the ability to access them in this simple way..